strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/maven/MavenArtifactControllerTest.java

```diff
@@ -841,15 +841,39 @@ public void testThatMavenProxyRepoDoesNotBlockTheConnectionPool()
                .get(url)
                .peek()
                .then()
-               .statusCode(HttpStatus.NOT_FOUND.value());
+               .statusCode(HttpStatus.OK.value());
+    }
+
+    @Test
+    public void shouldDownloadProxiedSnapshotArtifact()
+            throws Exception
+    {
+        String url = getContextBaseUrl() +
```

Comment:
Perhaps we could have some test artifacts deployed to a `snapshot` repository in strongbox that is included in a group through which the test case can resolve it directly against strongbox itself...? This should be more reproducible.